### PR TITLE
fix: widget clone issue

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-11970](https://app.clickup.com/t/2645698/DEV-11970)

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-11970 Widget cloning is not working](https://app.clickup.com/t/2645698/DEV-11970)

File: domains/grid/components/GridFloatingMenu.vue [hunk 1]
```diff
@@ -137,7 +138,7 @@ onUnmounted(() => {
 
 <template>
   <div
-    v-if="isConnected"
+    v-if="isConnected && isConnectedUserViewingOwnProfile"
```
Comment (JeneaVranceanu) [inline domains/grid/components/GridFloatingMenu.vue:141]: LGTM, I only suggest renaming `isConnectedUserViewingOwnProfile` property to a shorter version if there's no other conflicting context that could be misleading. I think `isViewingOwnProfile` carries the same meaning and is shorter.
```suggestion
    v-if="isConnected && isViewingOwnProfile"
```

**Review by JeneaVranceanu — APPROVED** [review]

File: domains/grid/components/GridFloatingMenu.vue (hunk 1 above)
Comment (dzbo) [reply domains/grid/components/GridFloatingMenu.vue:141]: I like descriptive names even that they seem to long so just by looking on them you get exact context. I'm fine for `isViewingOwnProfile` although it's not straight what type of "viewer" we deal here.

**Review by doubleppereira — APPROVED** [review]
LGTM but you will need to update the snapshots

**Comment by Hugoo** [comment]
Task linked: [DEV-11991 Bug: we see the edit mode on other UP](https://app.clickup.com/t/2645698/DEV-11991)

**Pull request merged**